src/widgets/unfold_widgets/preprocessing_widget.py

```diff
@@ -154,7 +161,7 @@ def _clear_plot(self):
 
     def add_columns_to_layout(self):
         self.clear_column_layout()
-        columns = self.engine.get_raw_columns()
+        columns = self.engine.get_columns()
```

Comment:
We lose access to raw_data from preprocessing section. We should discuss if we need raw_data and how get them from preprocessing section.

Reply:
This is also connected with new column names. We should discuss if we need old dimension names after a reduction.